sql-statements/sql-statement-flashback-to-timestamp.md

```diff
@@ -15,6 +15,16 @@ TiDB v6.4.0 introduces the `FLASHBACK CLUSTER TO TIMESTAMP` syntax. You can use
 
 </CustomContent>
 
+<CustomContent platform="tidb">
+
+> **Warning:**
+>
+> When you use this feature in TiDB v7.1.0, some Regions might not be finished after the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).
```

Comment:
```suggestion
> When you use this feature in TiDB v7.1.0, some Regions might remain in the FLASHBACK process even after the completion of the FLASHBACK operation. It is recommended to avoid using this feature in v7.1.0. For more information, see issue [#44292](https://github.com/pingcap/tidb/issues/44292).
```
@Defined2014 PTAL